packages/pigeon/lib/src/pigeon_lib.dart

```diff
@@ -274,7 +275,6 @@ class PigeonOptions {
     this.gobjectOptions,
     this.dartOptions,
     this.copyrightHeader,
-    this.oneLanguage,
```

Comment:
Doesn't this mean the PR needs to be a major version change, not a bugfix change? Unless I'm missing something, this whole file is exported, and the class is public.